css/app.css

```diff
@@ -1,11 +1,13 @@
 :root {
+  --dark-accent-color: #3a3a3a;
   --dark-background-color: #1f1f1f;
   --dark-divider-color: rgb(255, 255, 255, 0.12);
   --dark-highlight-color: rgb(95, 99, 104);
   --header-height: 48px;
+  --light-accent-color: rgb(248, 249, 250);
```

Comment:
Nit: I would advocate for using hex codes everywhere if you can?

Reply:
chromium style prefers rgb: https://chromium.googlesource.com/chromium/src/+/master/styleguide/web/web.md#color